src/test/groovy/org/liquibase/groovy/delegate/ChangeSetMethodTests.groovy

```diff
@@ -349,7 +349,8 @@ ALTER TABLE monkey_table DROP COLUMN angry;"""
             addColumn(tableName: 'monkey') {
                 column(name: 'diet', type: 'varchar(30)')
             }
-            rollback(rollbackId: CHANGESET_ID, changeSetAuthor: CHANGESET_AUTHOR)
+            // rollbackId is invalid attribute
+            rollback(changeSetId: CHANGESET_ID, rollbackId: CHANGESET_ID, changeSetAuthor: CHANGESET_AUTHOR)
```

Comment:
I added `changeSetId` because values other than `rollbackId` need to be standard values.